xml_converter/src/category.hpp

```diff
@@ -9,36 +9,33 @@
 #include "rapidxml-1.13/rapidxml.hpp"
 #include "trail.hpp"
 
-using namespace std;
 
 
+using namespace std;
 
-#define PARSEABLE_VAR(...) CLASS_PARSEABLE_VAR(Category, __VA_ARGS__ )
-#define PARSEABLE_SUBVAR(...) CLASS_PARSEABLE_SUBVAR(Category, __VA_ARGS__ )
+// #define PARSEABLE_VAR(...) CLASS_PARSEABLE_VAR(Icon, __VA_ARGS__ )
```

Comment:
You commented these out so it does not matter, but "Category" was changed to "Icon". I'm guessing this is statically written as "Icon" in the template file. You can ignore this because it is going to get removed in the next iteration.